src/libPMacc/include/algorithms/math/doubleMath/floatingPoint.tpp

```diff
@@ -60,6 +72,29 @@ struct Float2int_rd<double>
     }
 };
 
+template<>
+struct Float2int_rn<double>
+{
+    typedef int result;
```

Comment:
I think @psychocoderHPC is the right person to review this PR due to the PMacc interfaces that one should know when evaluating it.